internal/pkg/log/log.go

```diff
@@ -50,21 +51,24 @@ func Init(configLogLevel string, logFile string, logInConsole bool) error {
 		return err
 	}
 
-	f, err = os.OpenFile(logPath, os.O_CREATE|os.O_WRONLY|os.O_APPEND, 0777)
-	if err != nil {
-		return err
+	fileWriter := &lumberjack.Logger{
+		Filename:   logPath,
+		MaxSize:    logFileSize, // megabytes
+		MaxBackups: 1,
+		MaxAge:     0,     //days
+		Compress:   false, // disabled by default
 	}
+	writers = append(writers, fileWriter)
 
 	if logInConsole {
 		consoleWriter := zerolog.ConsoleWriter{Out: os.Stdout}
-		multi = zerolog.MultiLevelWriter(f, consoleWriter)
-	} else {
-		multi = zerolog.MultiLevelWriter(f)
+		writers = append(writers, consoleWriter)
```

Comment:
Is there any specific reason to use writers array? We can add file writer and console writer together using zerolog.MultiLevelWriter(fileWriter,consoleWriter).